.changeset/empty-garlics-smoke.md

```diff
@@ -0,0 +1,37 @@
+---
+'@primer/react-brand': minor
+---
+
+- Added a new `RiverAccordion` component.
+
+  Usage example:
+
+  ```tsx
+  <RiverAccordion>
+    <RiverAccordion.Item>
+      <RiverAccordion.Heading>A Heading</RiverAccordion.Heading>
+      <RiverAccordion.Content>Some content</RiverAccordion.Content>
+      <RiverAccordion.Visual>
+        <img src="placeholder.png" alt="placeholder" />
+      </RiverAccordion.Visual>
+    </RiverAccordion.Item>
+
+    <RiverAccordion.Item>
+      <RiverAccordion.Heading>Another Heading</RiverAccordion.Heading>
+      <RiverAccordion.Content>Some content</RiverAccordion.Content>
+      <RiverAccordion.Visual>
+        <img src="placeholder.png" alt="placeholder" />
+      </RiverAccordion.Visual>
+    </RiverAccordion.Item>
+
+    <RiverAccordion.Item>
+      <RiverAccordion.Heading>One More Heading</RiverAccordion.Heading>
+      <RiverAccordion.Content>Some content</RiverAccordion.Content>
+      <RiverAccordion.Visual>
+        <img src="placeholder.png" alt="placeholder" />
+      </RiverAccordion.Visual>
+    </RiverAccordion.Item>
+  </RiverAccordion>
+  ```
+
+- Fixed a bug in the `Heading` component where the component would be re-mounted each render, causing it to lose focus when used inside a focusable element.
```

Comment:
It doesn't seem like the user needs to make any changes here but double checking that's the case @joshfarrant? There's no breaking changes here?

Reply:
Yes correct, no changes needed for consumers, just a edge-case bug fix